src/components/map/Markers.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import { StatusColor, MarkerTypes } from './Map.constants';
+
+export default ({ type, status }) => {
+  const Marker = MarkerTypes[type];
```

Comment:
`Marker` variable name is confusing here. Maybe use something like `ReactSVGIcon`?

Reply:
Done.